### ISSUE TITLE
[1.12] Plant Sower and seed filter spread

### ISSUE THREAD

**Issue by DeadSix27** [Buuz135/Industrial-Foregoing#72]
Industrial Foregoing Version:  newest at the time
TeslaCoreLib Version: newest at the time

If you have a 3x3 area and put more than 1 seed in the top left corner, it will plant that seed right there and .. thats it. Is there a way to just spread all the seeds out across the entire area?
Now you might say, just lock the seed into each slot, but that would mean each slot has to be filled with seeds at all times.. but that isn't possible with plants that do not produce new seeds, e.g you'll have to recollect the ones that get harvested and replant them, however upon re-inputting them they all get sent to slot no.1 resulting in the 3x3 area going down to 1 block.

tl;dr Is it possible to disable the filter and just let the seeds spread across 3x3?

**Comment by Buuz135** [comment]
Use the Item Splitter

**Comment by DeadSix27** [comment]
@Buuz135 And where does that block output them to? I just inserted 64 seeds into it, with a sower next to it and now those seeds are just.. gone.

**Comment by Buuz135** [comment]
I will have to fix it but the Plant sower will only hold 16 items its not a big of an issue that they are not even

**Comment by DeadSix27** [comment]
@Buuz135 What i mean is, they do not even appear, is that an invisible inventory?

**Comment by Buuz135** [comment]
No, they get voided

**Comment by faceofcat** [comment]
did you buy that machine from the Shady Merchant?... :D

**Comment by Buuz135** [comment]
Im the shady one!
